adafruit_pn532/adafruit_pn532.py

```diff
@@ -447,7 +474,7 @@ def mifare_classic_read_block(self, block_number):
         # Return first 4 bytes since 16 bytes are always returned.
         return response[1:]
 
-    def mifare_classic_write_block(self, block_number, data):
+    def mifare_classic_write_block(self, block_number: int, data: bytearray) -> bool:
```

Comment:
`data` can be any readable array, so let's annotate it as `circuitpython_typing.ReadableBuffer`.